npm/react18/src/index.ts

```diff
@@ -53,6 +52,7 @@ export function mount (jsx: React.ReactNode, options: MountOptions = {}, rerende
   // to wipe away any state
   cleanup()
   const internalOptions: InternalMountOptions = {
+    // @ts-expect-error
```

Comment:
this has to do with a type mismatch now that exists in the root of the monorepo 🥴 . This will be addressed with #29607